src/tslint.json

```diff
@@ -1,6 +1,12 @@
 {
-    "extends": "../tslint.json",
+    "extends": ["../tslint.json","angular-tslint-rules"],
     "rules": {
+        "no-unnecessary-class": [
+            true,
+            "allow-constructor-only",
+            "allow-static-only",
+            "allow-empty-class"
+        ], 
```

Comment:
the rules are there for a reason, they are hints on how to improve your code, try returning the rules and fixing the linting errors.